_shows/19_20/cinnamon_tea.md

```diff
@@ -0,0 +1,79 @@
+---
+
+title: Cinnamon Tea
+playwright: Beth White
+student_written: true
+period: Spring
+season: Online
+season_sort: 390
+date_start: 2020-08-23
+date_end: 2020-08-23
+venue: Online (via YouTube)
```

Comment:
We should standardise what our online venue is and update the rest accordingly.

Reply:
What are your thoughs on the name for this venue?

Reply:
I wonder if we do 
```yml
venue: YouTube
venue_sort: Online
```
To leave it open for shows hosted elsewhere?

Reply:
Further to this: where do the online TheSpace shows go? Under online or under TheSpace UK? 🤔

Reply:
I'd go TheSpaceUK